float32 throttle
float32 steer
int8 gear
float32 brake
uint8 cmd
